src/test/java/io/airlift/units/TestDataSize.java

```diff
@@ -32,28 +32,53 @@
 import static io.airlift.units.DataSize.succinctBytes;
 import static io.airlift.units.DataSize.succinctDataSize;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.offset;
-import static org.assertj.core.data.Percentage.withPercentage;
+import static org.assertj.core.api.Assertions.within;
 import static org.testng.Assert.assertEquals;
 
 public class TestDataSize
 {
     @Test
     public void testSuccinctFactories()
     {
-        assertEquals(succinctBytes(123), new DataSize(123, BYTE));
-        assertEquals(succinctBytes((long) (5.5 * 1024)), new DataSize(5.5, KILOBYTE));
-        assertEquals(succinctBytes(5 * 1024 * 1024), new DataSize(5, MEGABYTE));
+        DataSize oneHundred = DataSize.ofBytes(123);
```

Comment:
This variable name is misleading. I think inlining here is clearer:
```java
assertEquals(succinctBytes(123), DataSize.ofBytes(123));
assertEquals(succinctBytes(123).getUnit(), BYTE);

Reply:
Changed.